pkg/acceptance/helpers/user_client.go

```diff
@@ -100,12 +118,19 @@ func (c *UserClient) SetDaysToExpiry(t *testing.T, id sdk.AccountObjectIdentifie
 	require.NoError(t, err)
 }
 
-func (c *UserClient) SetType(t *testing.T, id sdk.AccountObjectIdentifier, value string) {
+func (c *UserClient) SetType(t *testing.T, id sdk.AccountObjectIdentifier, userType sdk.UserType) {
+	t.Helper()
+	ctx := context.Background()
+
+	_, err := c.context.client.ExecForTests(ctx, fmt.Sprintf("ALTER USER %s SET TYPE = %s", id.FullyQualifiedName(), userType))
```

Comment:
Why is this not a part of the SDK?

Reply:
It is, it was not the part before, that's why it was handled through exec. Replaced.

But the unset was really missing in the SDK, so added it with tests and replaced too.